.editorconfig

```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
```

Comment:
Andy I love your code! I always learn something new looking at it! Good job!

Reply:
Thank you Angel. I too like to read other student's implementations of Lambda projects. Everyone has their own way of coding the solution. For this project I tried to bring in the MVC (Model-View-Controller) pattern I use with client code.